tech.json

```diff
@@ -340,7 +343,7 @@
               "requires": [
                 "canWalljump"
               ],
-              "note" : "Climbing a wall with several consecutive walljumps.",
+              "note" : "Climbing a wall with three or more consecutive walljumps, where a single failure will require restarting.",
```

Comment:
Failure is a strange descriptor since missing a walljump means you fall a bit and try again a little lower.  Accidently breaking spin could be the failure at this skill level; an enemy hit doesn't seem meant to be included in this meaning.